v4/source/definitions.yaml

```
@@ -2236,6 +2236,36 @@ definitions:
         type: object
         description: Settings schema used to define the System Console UI for the plugin.
 
+  MarketplacePlugin:
+    type: object
+    properties:
+      homepage_url:
+        type: string
+        description: "URL that leads to the homepage ot the plugin."
+      icon_data:
+        type: string
+        description: "Base64 that represents icon of the plugin."
+      download_url:
+        type: string
+        description: URL to download the plugin.
+      release_notes_url:
+        type: string
+        description: URL that leads to the release notes of the plugin.
+      labels:
+        type: array
+        items:
+          type: string
+        description: A list of the plugin labels.
+      signature:
+        type: string
+        description: "Signature of the plugin"
```

Any particular reason for this to be in quotes?
```suggestion
        description: Signature of the plugin
```